packages/components-js/tests/vrt/playwright/specs/states/select.vrt.ts

```diff
@@ -0,0 +1,151 @@
+import { expect, type Page, test } from '@playwright/test';
+import { schemes, themes, viewportWidthM } from '@porsche-design-system/shared/testing/playwright.vrt.config';
+import {
+  forceFocusHoverState,
+  forceFocusState,
+  forceHoverState,
+  getPlaygroundPseudoStatesMarkup,
+  type PrefersColorScheme,
+  setContentWithDesignSystem,
+} from '../../helpers';
+import { type Theme } from '@porsche-design-system/utilities-v2';
```

Comment:
Would be safer/better to import from components since this is somewhat unrelated.